packages/fxa-shared/metrics/navigation-timing.ts

```diff
@@ -13,29 +16,80 @@
 
 const NAV_ENTRY_TYPE = 'navigation';
 
+export const checkNavTiming = (navTimings: any) => {
+  const errors = [];
+  function check(key: string, min?: number, rel?: string) {
+    if (navTimings[key] == null) {
+      errors.push(
+        new InvalidNavigationTimingError(
+          `navTiming.${key}`,
+          'missing',
+          navTimings[key],
```

Comment:
I think I might be confused here.

We only get into this block if `navTimings[key]` is `null` (per L22 above), and the signature of `InvalidNavigationTimingError()` is roughly:

```js
export class InvalidNavigationTimingError extends Error {
  constructor(
    name: string,
    violation: string,
    critical: boolean,
    val: string | number | null | undefined,
    limitOrDefault?: string | number
  ) { … }
}
```

Does that mean `navTimings[key]` values are all Booleans (since `critical: boolean` is the third arg), or is `navTimings[key]` an object w/ string|number|null|undefined values and we're missing the `critical` arg?

Reply:
No, this is a mistake on my part. I refactored this to add a 'critical' flag, and missed this one. Thanks for catching this!